### ISSUE TITLE
Watir::Wait.until and while lose default timeout value in error message

### ISSUE THREAD

**Issue by abaird** [watir/watir-webdriver#354]
Note the following results where the timeout message loses the value of the timeout timer:
```
irb(main):001:0> require 'watir-webdriver' 
=> true                                                       
irb(main):002:0> Watir::Wait.until {false}   
Watir::Wait::TimeoutError: timed out after  seconds  
        from /Users/alanbaird/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/watir-webdriver-0.7.0/lib/watir-webdriver/wait.rb:4
1:in `until'               
        from (irb):2     
        from /Users/alanbaird/.rbenv/versions/2.1.6/bin/irb:11:in `<main>'                                                     
irb(main):003:0> Watir.default_timeout 
=> 30                                                     
irb(main):004:0> Watir::Wait.while {true}
Watir::Wait::TimeoutError: timed out after  seconds 
        from /Users/alanbaird/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/watir-webdriver-0.7.0/lib/watir-webdriver/wait.rb:5
7:in `while'             
        from (irb):4    
        from /Users/alanbaird/.rbenv/versions/2.1.6/bin/irb:11:in `<main>'   
```

This happens because inside the Watir::Wait class, the value of timeout is set to Watir.default if there is no value specified in the call to until or while. However, this only happens as a local variable assignment in Watir::Wait.run_with_timer. When message_for is called, timeout is still nil and thus you see no value when the TimeoutError message is printed.

I suggest moving `raise TimeoutError, message_for(timeout, message)` into Watir::Wait.run_with_timer. I don't quite have time right now to submit a test and PR for this as I am debugging at work. Hopefully I can get some time later on this week to do that.

**Comment by abaird** [comment]
I'm running version 0.7.0 of watir-webdriver on Mac OSX.

**Comment by p0deje** [comment]
This has already been fixed in b902f8b8e2fb9ed7e2b5b83d049c8151d2cc002d, but it's only a part of 0.8.0 release. Please, upgrade.
